src/main/java/berry/task/Deadline.java

```diff
@@ -0,0 +1,46 @@
+package berry.task;
+
+import berry.exception.IncorrectDateException;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a deadline task.
+ */
+public class Deadline extends Task {
+
+    /** The date to do the task by */
+    protected LocalDate by;
```

Comment:
any reason why u use protected instead of private, since deadline doesnt have a subclass?

Reply:
I think I was just not paying extra attention to the access mods. Thanks for pointing it out! I will make sure to double-check them <: